cmd/submit.go

```diff
@@ -202,7 +202,15 @@ figuring things out if necessary.
 			return err
 		}
 
-		fmt.Fprintf(Out, "Submitted. View at %s\n", solution.URL)
```

Comment:
Should this line go in the `else` block below? 
I may be missing something but otherwise will there be no feedback for a submitted solution that has not been auto-approved? @kytrinyx @nywilken

Reply:
@robphoenix Yes, I probably should have commented about this. I'll change it to whatever we conclude here. Thanks for the review!